src/sniffer.c

```diff
@@ -3498,10 +3504,44 @@ int ssl_DecodePacket(const byte* packet, int length, byte** data, char* error)
     ret = ProcessMessage(sslFrame, session, sslBytes, data, end, error);
     if (RemoveFatalSession(&ipInfo, &tcpInfo, session, error)) return -1;
     CheckFinCapture(&ipInfo, &tcpInfo, session);
+
+    /* Pass back Session Info after we have processed the Server Hello. */
```

Comment:
Let's put this block in its own function.

Reply:
Will do. I'm going to add a trace function for this SSLInfo structure as well.